projects/netgrif-components/src/lib/data-fields/text-field/textarea-field/textarea-field.component.html

```diff
@@ -7,11 +7,14 @@
               #textArea
               [placeholder]="dataField.placeholder"
               [required]="dataField.behavior.required"
-              [formControl]="formControlRef"
+              [(ngModel)]="dataField.value"
```

Comment:
why? A lot of functionality (e.g. enabling/disabling) depends on the `FormControl` as far as I know.

I notice the hidden `<input>` on line 17. Does it work, when for example disabling the hidden field?

Reply:
@camperko I think you can answer this.